packages/components/pie-cookie-banner/vite.config.js

```diff
@@ -1,3 +1,15 @@
 import viteConfig from '@justeattakeaway/pie-components-config/vite.config';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
 
-export default viteConfig;
+export default viteConfig({
+    plugins: [
+        viteStaticCopy({
+            targets: [
+                {
```

Comment:
This copy is always done _before_ the component is built, so references to `dist/locales` in the index.ts work as we'd expect 👍🏼